src/overlays/actors/ovl_En_Fr/z_en_fr.c

```diff
@@ -8,14 +9,169 @@ void EnFr_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnFr_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnFr_Update(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_05002470;
-extern UNK_TYPE D_050036F0;
-extern UNK_TYPE D_060007BC;
-extern UNK_TYPE D_060011C0;
-extern UNK_TYPE D_06001534;
-extern UNK_TYPE D_0600B498;
+void EnFr_UpdateIdle(EnFr* this, GlobalContext* globalCtx);
+void EnFr_UpdateActive(EnFr* this, GlobalContext* globalCtx);
+
+void EnFr_IsDivingIntoWater(EnFr* this, GlobalContext* globalCtx);
+void EnFr_DivingIntoWater(EnFr* this, GlobalContext* globalCtx);
```

Comment:
we only need to forward declare functions when necessary. DivingToWater at least is not needed as the definition comes before any usages, should check the others as well